scivision/catalog/catalog.py

```diff
@@ -19,8 +15,6 @@ def _validate_query(query: dict) -> dict:
 
 
 class BaseCatalog(abc.ABC):
-    def __init__(self):
-        self._uri = CATALOG_URI
 
     @property
     def URI(self):
```

Comment:
Should this be removed too?

Reply:
I've removed simply because it no longer does anything, but we can always add another init func to this class in future if it makes sense to do so